### PR TITLE
Fuj/refact diopi test

### PR DESCRIPTION
**Pull request by jingguo-st**
## Motivation and Context

refact diopi_test

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: diopi_test/python/conformance/__init__.py [hunk 1]
```diff
@@ -1,11 +1,18 @@
 # Copyright (c) 2023, DeepLink.
-from .diopi_runtime import Tensor, Dtype, Device, raw_like, diopiTensor, Context
+import os
+import sys
+
+from diopilib import diopiTensor, Context, Device, Dtype
```
Comment (yangbofun) [inline diopi_test/python/conformance/__init__.py:5]: 这一行要删除，gen_data时只依赖pytorch，不依赖我们是否编译diopi

**Comment by yangbofun** [comment]
diopi_test/python/tests内的测试，分一下类，一个是测的diopi的测试，一个是测试diopi_test组建的测试。

File: diopi_test/scripts/ci_script.sh [hunk 2]
```diff
@@ -3,9 +3,9 @@ set -e
 
 case $1 in
   py-lint)
-    (echo "py-lint" && flake8 --ignore=E501,F841 python/conformance/diopi_functions.py \
-       && flake8 --ignore=E501,F401 --exclude=python/conformance/diopi_functions.py,scripts/cpplint.py,impl/,third_party/,python/conformance/model_config/ \
-       && flake8 --ignore=E501,F401 python/conformance/model_config/process_ops.py python/conformance/model_config/generate_config.py ) \
+    (echo "py-lint" && flake8 --ignore=E501,F401,F841 python/conformance/diopi_functions.py \
```
Comment (yangbofun) [inline diopi_test/scripts/ci_script.sh:6]: 这个文件是检查的规则，尽量不要改动。修改代码报错的地方。

File: diopi_test/python/conformance/__init__.py (hunk 1 above)
Comment (jingguo-st) [reply diopi_test/python/conformance/__init__.py]: decouple the depandency: done

**Review by zhangzefeng92 — APPROVED** [review]

**Review by LeungChiNan — APPROVED** [review]

**Review by yangbofun — APPROVED** [review]

**Pull request merged**